js/on-event.js

```diff
@@ -37,16 +37,17 @@ export const showMessage = function (parent, element) {
   parent.appendChild(element);
 
   const elementClickHandler = function () {
-    element.classList.add('visually-hidden');
+    element.remove();
     element.removeEventListener('click', elementClickHandler);
-  }
+  };
+
 
   const documentКeydownHandler = function (evt) {
```

Comment:
```suggestion
  const documentKeydownHandler = function (evt) {
```

K - кирилицей написана, поправь